networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -411,13 +411,14 @@ def bfs_layers(G, sources):
     G : NetworkX graph
         A graph over which to find the layers using breadth-first search.
 
-    sources : node in `G` or list of nodes in `G`
-        Specify starting nodes for single source or multiple sources breadth-first search
+    sources : node in `G` or iterable of nodes in `G`
+        Specify starting nodes for single source or multiple sources
+        breadth-first search.
 
     Yields
     ------
-    layer: list of nodes
-        Yields list of nodes at the same distance from sources
+    layers: list of nodes
```

Comment:
```suggestion
    layer : list of nodes
```
The name and type of values in "Yields" section should refer to a _single_ item that is yielded: https://numpydoc.readthedocs.io/en/latest/format.html#yields